content/news/2025-07-28-collab-clermont/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: 2025 Fink Collaboration Meeting
+date: 2025-07-28
+cardimage: lpca2.jpeg
+---
+The 2025 Fink Collaboration Meeting happened from 16 to 18 July 2025, in Clermont Ferrand - France
+<!--more-->
+
+Part of the Fink community met in Clermont Ferrand, during a few hot summer days, to discuss the future of Fink and make preparations for the upcoming Rubin data.  
```

Comment:
Part of the Fink community [...] for the upcoming Rubin data. We were hosted by the [Société des Hôtels Littéraires](https://www.hotelslitteraires.fr/en/about-us/benefit-corporation/) which was also an opportunity to immerse ourselves in the world of books and literature.